### PR TITLE
[13.0] Add shopinvader_locomotive_wishlist

### PR DESCRIPTION
**Pull request by simahawk**
Addresses https://github.com/shopinvader/odoo-shopinvader/issues/574

### CODE REVIEW

File: shopinvader_locomotive_wishlist/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Export wihslist data to Locomotive user.
+
+This is module is mandatory if you want wishlists to work properly on Locomotive
```
Comment (TDu) [inline shopinvader_locomotive_wishlist/readme/DESCRIPTION.rst]:
```suggestion
This module is mandatory if you want wishlists to work properly on Locomotive
```

**Review by TDu — APPROVED** [review]

File: test-requirements.txt [hunk 2]
```diff
@@ -7,4 +7,6 @@ vcrpy
 vcrpy-unittest
 openupgradelib @ git+https://github.com/OCA/openupgradelib.git
 unittest2 # For shopinvader test_controller, which inherits component
-odoo-test-helper
+#odoo-test-helper
+# TMP dependency to include https://github.com/akretion/odoo-test-helper/pull/1
+git+git://github.com/simahawk/odoo-test-helper.git@master#egg=odoo-test-helper
```
Comment (sebalix) [inline test-requirements.txt]: Use https://github.com/akretion/odoo-test-helper/pull/7 instead?

**Review by sebalix — APPROVED** [review]
Pre-approving, one minor comment about the `odoo-test-helper` dependency

File: test-requirements.txt (hunk 2 above)
Comment (simahawk) [reply test-requirements.txt]: I had issues w/ it and my version is working fine in other PRs here and on https://github.com/OCA/search-engine/pull/50
It's just that nor me nor @sebastienbeau had the time to dig further in what is not handled by my version or his.

Comment (simahawk) [reply test-requirements.txt]: removed as new version has been released

**Pull request merged**